Adapt MediaRecorder API to new events structure (#13850)
mdn/content: files/en-us/web/api/mediarecorder/start_event/index.md

>   - Property

Reviewer: ```suggestion
  - Event
```
Or it won't be listed correctly in the sidebar.

  - Reference
  - Video
  - start
browser-compat: api.MediaRecorder.start_event
---
{{APIRef("Media Recorder API")}}

> The `start` event is thrown as a result of the
> {{domxref("MediaRecorder.start()")}} method being invoked. At this point, the data

Reviewer 2: ```suggestion
The `start` event is fired when
{{domxref("MediaRecorder.start()")}} is called. At this point, the data
```
